plugins/repoclosure.py

```diff
@@ -86,9 +99,12 @@ def _get_unresolved(self, arch=None):
         if self.opts.check:
             checkpkgs = set()
             for repo in self.opts.check:
-                for pkgs_filtered in available.filter(reponame=repo):
+                for pkgs_filtered in self.base.sack.query().available().filter(reponame=repo):
```

Comment:
I think that this should be solved by option like   in yum "--newest" ("check only the newest packages in the repos"). But this option should be applied also to line 74: available = self.base.sack.query().available().filter(latest=True)